apps/server/lib/lexical/server/code_intelligence/completion/env.ex

```diff
@@ -490,4 +493,8 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Env do
         {:ok, tokens, ''}
     end
   end
+
+  defp maybe_trim_dot(character, line) do
+    if String.ends_with?(line, "."), do: character - 1, else: character
```

Comment:
This feels very specific to one case. Was it the only case that was missed? If it was, where's the unit test that shows the prior behavior and the fixed behavior?

Reply:
Ok, I deleted this function